CONTRIBUTING.md

```diff
@@ -166,86 +92,77 @@ conda activate cudf_dev
 ```
 - For other CUDA versions, check the corresponding cudf_dev_cuda*.yml file in conda/environments
 
-- Build and install `libcudf` after its dependencies. CMake depends on the `nvcc` executable being on your path or defined in `$CUDACXX`.
-```bash
-$ cd $CUDF_HOME/cpp                                                       # navigate to C/C++ CUDA source root directory
-$ mkdir build                                                             # make a build directory
-$ cd build                                                                # enter the build directory
-
-# CMake options:
-# -DCMAKE_INSTALL_PREFIX set to the install path for your libraries or $CONDA_PREFIX if you're using Anaconda, i.e. -DCMAKE_INSTALL_PREFIX=/install/path or -DCMAKE_INSTALL_PREFIX=$CONDA_PREFIX
-$ cmake .. -DCMAKE_INSTALL_PREFIX=$CONDA_PREFIX                           # configure cmake ...
-$ make -j                                                                 # compile the libraries librmm.so, libcudf.so ... '-j' will start a parallel job using the number of physical cores available on your system
-$ make install                                                            # install the libraries librmm.so, libcudf.so to the CMAKE_INSTALL_PREFIX
-```
+### Build cuDF from Source
 
-- As a convenience, a `build.sh` script is provided in `$CUDF_HOME`. To execute the same build commands above, run the script as shown below.  Note that the libraries will be installed to the location set in `$INSTALL_PREFIX` if set (i.e. `export INSTALL_PREFIX=/install/path`), otherwise to `$CONDA_PREFIX`.
+- A `build.sh` script is provided in `$CUDF_HOME`. Note that  CMake depends on the `nvcc` executable being on your path or defined in `$CUDACXX` and the libraries will be installed to the location set in `$INSTALL_PREFIX` if set (i.e. `export INSTALL_PREFIX=/install/path`), otherwise to `$CONDA_PREFIX`.
 ```bash
 $ cd $CUDF_HOME
-$ ./build.sh                                                              # To build both C++ and Python cuDF versions with their dependencies
+$./build.sh libcudf             
 ```
-- To build only the C++ component with the script
+- Many other librairies like `dask-cudf` and `cudf-kafka` can be installed with this script. For the complete list of librairies as well as details about the script usage, run the `help` command:
 ```bash
-$ ./build.sh libcudf                                                      # Build only the cuDF C++ components and install them to $INSTALL_PREFIX if set, otherwise $CONDA_PREFIX
+$ cd $CUDF_HOME
+$./build.sh --help            
```

Comment:
I think we can avoid repeating the `cd` command here, but please feel free to ignore this suggestion if you disagree :) 

```suggestion
$ ./build.sh --help            
```

Reply:
Yes, I thought so as we are in the same section. But I just kept this style as it was.